### PR TITLE
light edits

### PR DESCRIPTION
**Pull request by jverzani**
In reading over the excellent documentation I noticed a few typos.

### CODE REVIEW

**Review by davibarreira — APPROVED** [review]

File: docs/src/tutorials/7_composite.md [hunk 1]
```diff
@@ -177,8 +177,9 @@ example of this would be parallel coordinate plots. The difficulty in creating s
 visualizations involves having to coordinate distict scales in order.
 
 Our example is inspired in one of the visualizations presented in the
-paper "Dece: Decision explorer with counterfactual explanations for machine learning models"
-(2020, heng, Furui and Ming, Yao and Qu, Huamin).
+paper ["Dece: Decision explorer with counterfactual explanations for machine learning models"](
+https://doi.org/10.48550/arXiv.2008.08353)
```
Comment (davibarreira) [inline docs/src/tutorials/7_composite.md:181]: This seems to be causing an error in the Documenter.

**Review by davibarreira — COMMENTED** [review]
Thanks a lot for the review! I'm really glad for the thorough review of the documentation. Hopefully you find the package useful.

It seems that one of the changes caused an error in the Documenter.

**Comment by jverzani** [comment]
I'll remove it. I had the link handy from looking up the paper, so thought it might be useful.

**Comment by davibarreira** [comment]
> I'll remove it. I had the link handy from looking up the paper, so thought it might be useful.

Thanks! It would be nice to have the link, but it seems that Documenter.jl has some issue with external links. I tried figuring out how to fix it, but I found no solution yet.

If I find it, I can then update the docs again with the link. Once more, thanks for all the corrections! I really apprettiate it.

**Comment by jverzani** [comment]
Thanks so much for this package. I've been having fun learning it and really appreciate the tutorials you put together!

**Comment by davibarreira** [comment]
Great to have feedback. Let me know if you find problems or have any suggestions.

**Pull request merged**